README.md

```diff
@@ -54,7 +54,12 @@
 ### Key Features <a name="key-features"></a>
 
```

Comment:
- [ ] Please make sure that 'Add new category' and 'Add new transaction' buttons match with the below design as requested in the project requirements to follow the design guidelines.

### Your design

![image](https://github.com/martinkarugaba/budget_app/assets/83803414/87f2bd0b-8ece-423e-be3d-61d06807e204)

### Given design 

![image](https://github.com/martinkarugaba/budget_app/assets/83803414/f5a83975-bf5a-4189-8dba-ade145cad63f)